packages/amplify-category-function/src/provider-utils/awscloudformation/index.ts

```diff
@@ -95,8 +95,10 @@ export async function updateResource(context, category, service, parameters, res
     context.amplify.updateamplifyMetaAfterResourceUpdate(category, answers.resourceName, 'dependsOn', result.dependsOn);
   }
 
+  // not a good implementation, will refractor when runtimePR changes will happen
   if (answers.parameters) {
     createParametersFile(context, answers.parameters, answers.resourceName);
+    createParametersFile(context, answers.parameters, answers.resourceName, 'parameters.json');
```

Comment:
since the functions parameters file is for checking permissions granted to other resources. Cron parameters cant go in that file. I have update to pick only parameters required for parameters.json file